proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -294,6 +293,22 @@ message WorkflowExecutionState {
     temporal.api.enums.v1.WorkflowExecutionStatus status = 4;
     VersionedTransition last_update_versioned_transition = 5;
     google.protobuf.Timestamp start_time = 6;
+
+    // Request IDs that are attached to the workflow execution.
+    // They can be from StartWorkflowExecution requests with conflict policy USE_EXISTING and
+    // conflict options to attached completion callbacks.
+    WorkflowExecutionRequestIDs attached_request_ids = 7;
```

Comment:
@bergundy With this, we can slowly deprecate the `create_request_id`, but I'm gonna do in a separate PR